examples/cryptocurrency-advanced/backend/src/transactions.rs

```diff
@@ -96,6 +102,10 @@ impl Transaction for Transfer {
         let hash = self.hash();
         let amount = self.amount();
 
+        if from == to {
+            Err(Error::SendingFundsToYourself)?
```

Comment:
Are you sure that we should check it in `execute`, not in `verify`? In basic cryptocurrency we do it in verify.